src/main/java/homeworks/christian/thirdAssigment/consumerInterface/Persona.java

```diff
@@ -0,0 +1,23 @@
+package homeworks.christian.thirdAssigment.consumerInterface;
+
+public class Persona {
+
+    private String nombre;
+
+    /**
```

Comment:
there is no need of javadoc for getter and setter unless they have an odd behavior